src/Microsoft.AspNetCore.OData/Batch/ODataBatchReaderExtensions.cs

```diff
@@ -227,6 +242,23 @@ private static HttpContext CreateHttpContext(HttpContext originalContext)
             context.Request.Cookies = originalContext.Request.Cookies;
             foreach (KeyValuePair<string, StringValues> header in originalContext.Request.Headers)
             {
+                var headerKey = header.Key.ToLowerInvariant();
```

Comment:
The style in this file/project seems to prefer ```type``` over ```var```, need to be consistent. If it were a new class then the ```var``` could be internally consistent in that file but here will need to adhere to existing standards.

Reply:
Coding guidelines are to use the actual type, not var.  Existing WebAPI doesn't adhere to this very well, but it's still good practice.

---
In reply to: [383750782](https://github.com/OData/WebApi/pull/2064#discussion_r383750782) [](ancestors = 383750782)